sdk/python/symbolchain/symbol/MessageEncoder.py

```diff
@@ -57,3 +62,36 @@ def encode(self, recipient_public_key: PublicKey, message: bytes):
 
 		tag, initialization_vector, cipher_text = encode_aes_gcm(SharedKey, self.key_pair, recipient_public_key, message)
 		return b'\1' + tag + initialization_vector + cipher_text
+
+	def try_decode_deprecated(self, recipient_public_key, encoded_message):
+		"""Tries to decode encoded wallet message, returns tuple:
+		* True, message - if message has been decoded and decrypted
+		* False, encoded_message - otherwise
+		"""
+
+		warnings.warn(
+			'This function is only provided for compatability with the original Symbol wallets.\n'
+			'Please use `try_decode` in any new code.',
+			category=DeprecationWarning,
+			stacklevel=2)
+
+		if 1 == encoded_message[0]:
+			try:
+				# wallet additionally hex encodes
+				return self.try_decode(recipient_public_key, bytes([1]) + unhexlify(encoded_message[1:].decode('utf8')))
+			except (UnicodeDecodeError, BinasciiError):
```

Comment:
what about `TypeError: Non-hexadecimal digit found` ?

Reply:
that is raised as `binascii.Error`